### PR TITLE
[PAYMENT] Implemented beta controller and service, added swagger to the controller

### PR DESCRIPTION
**Pull request by ivan0dyatlyukkk**

### CODE REVIEW

File: src/main/java/com/project/carsharingapp/controller/PaymentController.java [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    }
+
+    @PostMapping
+    @Operation(summary = "Create a new payment session")
+    public ResponseEntity<Object> create(
+            @RequestBody @Valid CreatePaymentSessionRequestDto requestDto
+    ) {
+        Payment payment = paymentService.create(requestDto);
+        URI url = URI.create(payment.getSessionUrl());
+        System.out.println(url);
```
Comment (OlegStashkiv) [inline src/main/java/com/project/carsharingapp/controller/PaymentController.java]: I think it's better to remove it

**Review by OlegStashkiv — APPROVED** [review]

File: src/main/java/com/project/carsharingapp/controller/PaymentController.java [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@Tag(name = "Payment controller", description = "Endpoint for managing payments")
```
Comment (DmytroV95) [inline src/main/java/com/project/carsharingapp/controller/PaymentController.java]:
```suggestion
@Tag(name = "Payment management", description = "Endpoint for managing payments")
```
To make a uniform style for controllers.

File: src/main/java/com/project/carsharingapp/dto/payment/CreatePaymentSessionRequestDto.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
+package com.project.carsharingapp.dto.payment;
+
+import jakarta.validation.constraints.NotNull;
+import lombok.Data;
+
+@Data
```
Comment (DmytroV95) [inline src/main/java/com/project/carsharingapp/dto/payment/CreatePaymentSessionRequestDto.java]:
```suggestion
@Getter
@Setter
```

File: src/main/java/com/project/carsharingapp/dto/payment/PaymentResponseDto.java [hunk 4]
```diff
@@ -0,0 +1,4 @@
+package com.project.carsharingapp.dto.payment;
+
+public class PaymentResponseDto {
```
Comment (DmytroV95) [inline src/main/java/com/project/carsharingapp/dto/payment/PaymentResponseDto.java:3]: What is the purpose of using empty class?

File: src/main/java/com/project/carsharingapp/service/impl/StripeService.java [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+import lombok.RequiredArgsConstructor;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import org.springframework.web.util.UriComponentsBuilder;
+
+@Service
+@RequiredArgsConstructor
+public class StripeService {
+    private static final String DEFAULT_URL_PART = "/api/payments";
+
```
Comment (DmytroV95) [inline src/main/java/com/project/carsharingapp/service/impl/StripeService.java]: Redundant line

File: src/main/java/com/project/carsharingapp/service/impl/StripeService.java [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+    private static final String SUCCESS_ENDPOINT = "success";
+    private static final String CANCEL_ENDPOINT = "cancel";
+    private static final Long STANDARD_QUANTITY_OF_RENTAL_CART = 1L;
+    private static final String DEFAULT_CURRENCY = "USD";
+    private final PaymentAmountHandlerStrategy handler;
```
Comment (DmytroV95) [inline src/main/java/com/project/carsharingapp/service/impl/StripeService.java:28]: Add whitespace between static and not static fields.

**Review by DmytroV95 — CHANGES_REQUESTED** [review]
Super! GJ! Left some minor comments.

**Review by OleksandrSaven — APPROVED** [review]
LGTM!

**Pull request merged**